lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -205,8 +205,8 @@ html.video-fullscreen {
         }
       }
 
-      span.proctored-exam-code {
-        margin-top: 5px;
+      input.proctored-exam-code {
```

Comment:
styling best-practice: don't depend on the specific type of element to which a styling class is applied. In this case it looks unnecessary (especially since you couldn't find the class applied previously? Did I understand that other comment right?)

Reply:
Sorry, my mistake. That comment belongs on the `proctored-exam-select-code` class.

But yes, totally agree about not depending on the element type. I'll fix that.